src/main/java/frc/robot/commands/ROSControl.java

```diff
@@ -31,6 +30,6 @@ public ROSControl(Drivetrain _drivetrain, ROS _ros) {
   // Called every time the scheduler runs while the command is scheduled.
   @Override
   public void execute() {
-    drivetrain.FlyWithWiresA(ros.getStarboardSpeed(), ros.getPortSpeed());
+    drivetrain.FlyWithWiresB(ros.getStarboardSpeed(), ros.getPortSpeed());
```

Comment:
Why change from A to B?

Reply:
Mode B is velocity control, ROS needs to use Velocity control.